pom.xml

```diff
@@ -223,5 +223,32 @@ limitations under the License.
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>release</id>
+      <build>
+        <pluginManagement>
```

Comment:
`pluginManagement` doesn't make sense here. You want the plugins goal to be executed when this profile is activated

Reply:
`mvn help:effective-pom -Pplexus-release` proofs that the plugin's goal *actually is* executed when the profile is activated (thanks to POM inheritance), so I do not understand your claim. Why do you think it is *not*? Or do you want to say that it is *bad style* to simply re-configure inherited configurations? Or do you want to be safe against the case that the plugin *could* be removed from the parent, and *then* it would effectively not execute anymore?

Reply:
Done as per your request.